pages/about.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Document</title>
+<link href="../styles/style.css" rel="stylesheet" />
 </head>
+
 <body>
-  
+    <div class="container">
+        <div class="header">
+        
+        </div>
+        <div class="sidebar">
+            <img class="img-side" src="../images/side.jpeg" alt="my image">
```

Comment:
The alt text should be more descriptive. People who are blind or have low vision depend on screen readers to tell them what is on the page. Screen readers use the alt text to describe the picture to the user so they know what is there. The words "my image" do not explain what this picture is to the user.